<launch>
    <arg name="device_id" default="motor_controller"/>
    <arg name="loop_rate" default="10"/>
    <node name="motor_controller" pkg="peripherals" type="motor_controller" respawn="true" respawn_delay="1" >
        <param name="device_id" value="$(arg device_id)"/>
        <param name="loop_rate" value="$(arg loop_rate)"/>
    </node>
</launch>
